testsuite/tracing.py

```diff
@@ -15,7 +15,7 @@ def __init__(self, jaeger_backend: bool, collector_url: str, query_url: str, cli
         self.client = client or KuadrantClient(verify=False)
         self.query = ApyProxy(query_url, session=self.client)
 
-    @backoff.on_predicate(backoff.fibo, lambda x: x == [], max_tries=7, jitter=None)
+    @backoff.on_predicate(backoff.fibo, lambda x: x == [], max_tries=5, jitter=None)
```

Comment:
why decrease it?

Reply:
`5` should be enough. I've done some testing.